### ISSUE TITLE
Are annotations designed to be co-authored or just exported?

### ISSUE THREAD

**Issue by lrosenthol** [readium/annotations#10]
When designing an annotation model around the Web Annots Data Model, one thing that needs to be considered is whether there is an expectation that more than one user can author a given annotation - or possibly "comment on a comment", or even just "like it".

This current design does not provide for those co-authoring/shared reviewing scenarios.  If that is what is intended, please feel free to close this issue.  however, if you do wish to support that use case, then there are bunch of changes required.

**Comment by llemeurfr** [comment]
I don't think that the W3C Web Annotation Data Model considers "annotations about annotations". 
With RDF everything is possible, but the target of an annotation is a `source` (web resource) plus `selector`, and the `selector` is a fragment of the `source`. I see no place for creating linked annotations here. 

I agree that the use case is interesting. 
I must warn that implementing this use case in reading systems would be highly complex (UX/UI + technical implementation).

**Comment by lrosenthol** [comment]
The Web Annot model fully supports commenting on comments.  IIRC there is even an example of how to do so.  It works by having the `target` of the annotation be another annotation (instead of a piece of text or other part of an asset).

But yes, doing so would add requirements for RS, no question.

This may not be something you need for v1 of your proposal, but I would think it should at least be tracked for the future.

**Comment by llemeurfr** [comment]
OK, I didn't see any sample of chained annotations in the W3C spec but this is feasible by design. 

As it would be a complex task for reading systems to support such a feature, and is something we don't see in the wild today, let's move this issue to a discussion and study it in the context of a V2 or level-aware specification.
